generators/python/src/fern_python/generators/sdk/client_generator/endpoint_function_generator.py

```diff
@@ -1107,13 +1109,19 @@ def generate_snippet(self) -> AST.Expression:
                 )
 
         if self.example.request is not None:
+            # For some reason the example type refernce is not marking it's type as optional, so we need to specify it so the
+            # snippets (and thus unit tests) write correctly
+            is_optional = False
+            if self.endpoint.request_body is not None and self.endpoint.request_body.get_as_union().type == "reference" and self.endpoint.request_body.get_as_union().request_body_type.get_as_union().type == "container" and self.endpoint.request_body.get_as_union().request_body_type.get_as_union().container.get_as_union().type == "optional":
+                is_optional = True
```

Comment:
nit: Might be worthwhile to put this in a helper, unless we plan to remove this check for something better later (re: your comment).

Reply:
updated!